index.js

```diff
@@ -55,64 +33,138 @@ app.post('/new-message', (req, res) => {
     ref.once('value').then(snapshot => {
       const chat = snapshot.val();
       const usernames = chat && chat.usernames ? chat.usernames : '';
-      console.log('le usernames', chat, usernames);
+
       if (chat && chat.message_id && chat.message_id > message.message_id) {
         console.log('discarting old message', chat.message_id, message.message_id);
-        return res.end('Error: message is undefined');
+        return res.end('discarting old message', chat.message_id, message.message_id);
```

Comment:
Minor,  but something like`Discarding... This is an older message`, sounds better.